pkgs/development/python-modules/e3-core/default.nix

```diff
@@ -24,26 +27,40 @@
 
 buildPythonPackage rec {
   pname = "e3-core";
-  version = "22.3.1";
+  version = "22.4.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "AdaCore";
-    repo = "e3-core";
+    repo = "${pname}";
```

Comment:
We're not reusing these, as they are technically something semantically different.

Reply:
Done